tests/python/gaia-ui-tests/gaiatest/tests/functional/marketplace/test_marketplace_addon.py

```diff
@@ -0,0 +1,87 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import By, Wait
+from gaiatest import GaiaTestCase
+from gaiatest.apps.base import Base, PageRegion
+from gaiatest.apps.marketplace.app import MarketplaceDev
+from gaiatest.apps.settings.app import Settings
+from gaiatest.apps.system.app import System
+
+
+class StatusBarAddon(PageRegion):
+    _root_locator = (By.CSS_SELECTOR, '#statusbar-maximized [id="statusbar-m"]')
+
+    def __init__(self, marionette):
+        marionette.switch_to_frame()
+        Base.__init__(self, marionette)
+        self.root_element = marionette.find_element(*self._root_locator)
+
+    @property
+    def is_displayed(self):
+        self.marionette.switch_to_frame()
+        # Workaround because is_displayed() doesn't work here
+        return (self.root_element.rect['x'] != 0 and
+                self.root_element.rect['y'] != 0 and
+                self.root_element.rect['width'] != 0 and
+                self.root_element.rect['height'] != 0)
+
+    @property
+    def text(self):
+        self.marionette.switch_to_frame()
+        # workaround because is_displayed doesn't work here
```

Comment:
Oh, I wasn't aware of that! That's worth to be mentioned in the code as a comment, I think